cli/kaos_cli/utils/decorators.py

```diff
@@ -138,8 +138,7 @@ def workspace_check(func):
     """
 
     def wrapper(*args, **kwargs):
```

Comment:
This `wrapper` function is broken and is causing the `kaos train deploy` command to fail. Lines [#151-152](https://github.com/KI-labs/kaos/pull/66/files#diff-594a140ded3666e5efbff16cddba3e4dL152-R152) should change to:

```
active_context = config['active']['environment']
base_url = config[active_context]['backend']['url']
current_workspace = config.get('pachyderm', 'workspace')
```

Reply:
This derivation for workspace name has been modified after the introduction of configObj